src/typegraph/core/src/runtimes/prisma/type_generation/snapshots/typegraph_core__runtimes__prisma__type_generation__test__update_many Post inp.snap

```diff
@@ -2,1712 +2,84 @@
 source: src/typegraph/core/src/runtimes/prisma/type_generation/mod.rs
 expression: tp.print(inp)
```

Comment:
@Natoandro I'm still surprised by the magnitude of this diff. I looked a the snapshot changes well and it makes sense and all the tests are also green but can you examine this file? I'm certain that the introduced changes should produce identical semantics but maybe we're losing some functionality by the lost types (which are mostly optionals from what I've seen. But maybe you'll catch something I didn't.